tests/integration/source_code/test_jobs.py

```diff
@@ -198,33 +202,48 @@ def test_lint_local_code(simple_ctx):
     assert len(problems) > 0
 
 
+@pytest.fixture(params=(DBFSPath, WorkspacePath))
+def make_directory_path(ws: WorkspaceClient, make_random: Callable[[int], str], request: pytest.FixtureRequest):
```

Comment:
I appreciate this is obscure, and I've removed it.

Pytest also allows [fixtures to be parameterised](https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#fixture-parametrize); tests that use the fixture will automatically be parameterised. It was a fixture here (instead of directly parameterising the test) to allow for cleanup.